src/components/Tables/MemberTasksTable.jsx

```diff
@@ -1,10 +1,48 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
 import classes from '../../pages/pages.module.css';
+import app from '../../firebs';
 import { Link } from 'react-router-dom';
-import { Button } from 'components/Buttons/Button/Button';
+import { useSelector } from 'react-redux';
+import {
+    getDatabase,
+    ref,
+    onValue,
+    set,
+    push,
+    child,
+    remove,
+    update,
+    query,
+    orderByChild,
+    equalTo,
+    get,
+} from 'firebase/database';
+import Button from 'react-bootstrap/Button';
 
-export const Table = ({ userTasks, params, linkPref }) => {
+export const Table = ({ userTasks, params, linkPref, userRole }) => {
+    /*eslint no-debugger: 1*/
+    const userID = useSelector((state) => state.user.id);
+    const database = getDatabase(app);
+    // const [statuses,setStatuses] = useState({});
+    // const updateItemStatus = (id,newStatus) => setStatuses({...statuses,[id]:newStatus})
+    const statusChange = (id, val, newStatus) => {
+        update(ref(database, 'tasks/' + id), {
+            name: val.name,
+            description: val.description,
+            startdate: val.startdate,
+            deadline: val.deadline,
+            assignedUsers: val.assignedUsers,
+            status: newStatus,
+        });
+    };
+    // useEffect(() => { console.log(statuses)}, [statuses])
```

Comment:
It is better to remove all the commented out code from the project if it does not contain an explanation.